internal/wasm/module.go

```diff
@@ -29,7 +29,7 @@ type DecodeModule func(
 ) (result *Module, err error)
 
 // EncodeModule encodes the given module into a byte slice depending on the format of the implementation.
-// See binary.EncodeModule
+// See binaryencoding.EncodeModule
 type EncodeModule func(m *Module) (bytes []byte)
```

Comment:
This looks like some sort of leftover. I can't really see any other references to this. Remove?

Reply:
if there are no references to something internal and the build passes, delete it.